package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gherkin-testcafe",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
There is no breaking change. Use v2.1.0.

Reply:
Ok, since TestCafe introduced a breaking change for the package. Updated as requested. Thanks.